ssn/ssn_separated/ssn.ttl

```diff
@@ -539,7 +459,7 @@ ssn:Latency rdf:type owl:Class ;
 ###  http://www.w3.org/ns/ssn/MaintenanceSchedule
 ssn:MaintenanceSchedule rdf:type owl:Class ;
                         rdfs:subClassOf ssn:OperatingProperty ;
-                        rdfs:comment "Schedule of maintenance for a system or sensor in the specified conditions." ;
+                        rdfs:comment "Schedule of maintenance for a System, Sensor or Actuator in the specified conditions." ;
```

Comment:
Schedule of maintenance for a System in the specified conditions.

(can also apply to Sampler)